### ISSUE TITLE
Bug: Vertex data panel shows up for glTexImage2D calls

### ISSUE THREAD

**Issue by sandy-8925** [apitrace/apitrace#517]
It would be nice if the panel showed the texture image based on the fuction arguments

**Comment by jrfonseca** [comment]
I agree it would be nice.  It's not straightforward though:

- the interpretation of the blob passed `glTexImage2D` depends on state other than the function parameters (e.g.,stride, alignment.), the GUI doesn't and can't keep track of that reliably, so we'd need to have a stride parameter in the texture image viewer too

- there are many different pixel formats, and the GUI actually doesn't have much code to interpret pixel formats -- so far we delegate that to the OpenGL driver via glretrace --, but to view the blobs straight from the gui we'd need to add code to convert pixelformats.

We could have the GUI ask glretrace to interpret the data.  But on the other hand, if one inspects state, one should be able to see the image that was uploaded to glTexImage2D.

In short, it's not a bad idea.  But I'm not sure it's worth it.  At any rate, I won't have the time to do it myself.
